### ISSUE TITLE
[Feature Request] I need to add/edit/delete activity

### ISSUE THREAD

**Issue by m50S79sM6SRNp8Jn** [frappe/gantt#338]

**Comment by m50S79sM6SRNp8Jn** [comment]
repository is not useful to me if I cannot manage activities

**Comment by farcloudy** [comment]
you could set a operate interface by custom_popup_html methon, so you can add delete or edit function by youself

**Comment by safwansamsudeen** [comment]
I agree with @farcloudy - this is a programming interface, not meant for end-users. @netchampfaris?

**Comment by netchampfaris** [comment]
@safwansamsudeen Agreed, but Gantt can provide APIs like `add_task`, `remove_task` and `edit_task` which developers can use to add functionality.

**Comment by safwansamsudeen** [comment]
@netchampfaris sounds good.

**Comment by safwansamsudeen** [comment]
We now have `update_task` - and the chart can easily be re-`render`ed after adding or remove tasks.
